radis/lbl/calc.py

```diff
@@ -401,6 +403,16 @@ def calc_spectrum(
     :py:class:`~radis.lbl.factory.SpectrumFactory`
     """
 
+    # path = join(radis.root_dir(), 'cache', f"{s_name}.spec")
+    path = join(root_dir(), "cache", "CO-hitran-700K-#5328.spec")
```

Comment:
/!\ this is hardcoded

Reply:
What you proposed as a naming convention is good to me ("molecule-database-T-###.spec"). There is a sufficient number of information (e.g. no need for pressure) and a random number to avoid overwriting.